README.md

```diff
@@ -223,13 +222,13 @@ by requiring the module by name.
 To install `aerospike` as a dependency of your project, in your project directory run:
 
 ```bash
-$ npm install aerospike
+npm install aerospike
 ```
```

Comment:
Is npm the recommended installation? Do you need to go through all the prerequisites first, or should most folks be able to run `npm install aerospike`? 

Keeping in line with our competitors and most other npm packages, we may want to move installation to the beginning of the readme and make the installation process more clearly defined. If `npm install aerospike` is all you need in most situations, we should highlight that.